src/Canvas2d/babylon.text2d.ts

```diff
@@ -0,0 +1,331 @@
+﻿module BABYLON {
+    export class Text2DRenderCache extends ModelRenderCache {
+        vb: WebGLBuffer;

```

Comment:
Yes, I understand, it's these kind of things that will be done asap, don't worry, I just wanted to get something out, but I fully understand it can't reasonably go live with dispose support. I'll do it tonight.